lib/services/fetch_basket.rb

```diff
@@ -0,0 +1,16 @@
+module Store
+  class FetchBasket
+    def call
+      BASKET.map do |item|
+      {
+        product: item.fetch_product,
+        id: item.id,
+        quantity: item.quantity,
+        price_with_vat: item.price_with_vat,
+        total_price: item.total_price,
+        total_price_with_vat: item.total_price_with_vat
+      }
```

Comment:
Here you are fetching product ID numbers, but then you don't use them in your basket view. Yes, you have a column named "ID" in there, but it's used for showing incremental numbers for basket items instead of product IDs (https://github.com/Kacper3331/day_four/pull/3/files#diff-0d148132bbbfa2709ec286e76b6528c8R22). I suggest changing one or the other.